src/test/node/node-runtime.test.ts

```diff
@@ -18,6 +18,7 @@ import { spawn, ChildProcess } from 'child_process';
 import Dap from '../../dap/api';
 import { delay } from '../../common/promiseUtil';
 import { nodeLaunchConfigDefaults, INodeLaunchConfiguration } from '../../configuration';
+import { sleep } from '../../int-chrome/testUtils';
```

Comment:
Don't use stuff from int-chrome, it might go away. This probably exists somewhere else

Reply:
There's `delay` in promise utils

Reply:
There's `delay` in promise utils

Reply:
Done